playbooks/coreos-bootstrap.yml

```diff
@@ -3,6 +3,7 @@
   gather_facts: False
   environment: "{{ proxy_env }}"
   roles:
+    - coreos_bootstrap
```

Comment:
this should go disabled by default with a toggle variable as we set it up on terraform for cloud providers